Userland/Libraries/LibGL/SoftwareGLContext.h

```diff
@@ -362,7 +367,9 @@ class SoftwareGLContext : public GLContext {
             decltype(&SoftwareGLContext::gl_tex_gen_floatv),
             decltype(&SoftwareGLContext::gl_fogf),
             decltype(&SoftwareGLContext::gl_fogfv),
-            decltype(&SoftwareGLContext::gl_fogi)>;
+            decltype(&SoftwareGLContext::gl_fogi),
+            decltype(&SoftwareGLContext::gl_lightf),
```

Comment:
`gl_materialf` and `gl_materialfv` should be reintroduced here, and they should get list support as well